marshmallow/fields.py

```diff
@@ -456,7 +462,9 @@ def deserialize(self, value):
         def do_deserialization():
             if value is missing:
                 if hasattr(self, 'required') and self.required:
-                    raise ValidationError('Missing data for required field.')
+                    message = self.required if isinstance(self.required, basestring) else \
```

Comment:
Please use parens rather than `\` for breaking long lines:

```python
                    message = (self.required if isinstance(self.required, basestring) else
                        'Missing data for required field.')
```